code/src/beetle.cpp

```diff
@@ -31,81 +31,163 @@ namespace r2d2::moving_platform {
             new_speed = 0;
         }
         qik_2s12v10_motorcontroller.set_m0_speed(new_speed);
-        qik_2s12v10_motorcontroller.set_m1_speed(-new_speed);
+        qik_2s12v10_motorcontroller.set_m1_speed(new_speed);
     } // namespace r2d2::moving_platform
```

Comment:
this comment suggests namespace r2d2::moving_platform ends here but it does not

Reply:
https://github.com/R2D2-2019/moving_platform/pull/30/commits/8070a9986ffcbc2c2bc39b0dd23e8bc5519e25c1